### DISCUSSION TITLE
Python Programlama Ders 9. Listeler | SonsuzUs Github

### DISCUSSION THREAD

**Discussion by giscus** [sonsuzus/sonsuzus.github.io#169] [Site]
# Python Programlama Ders 9. Listeler | SonsuzUs Github

liste sıralı değer kümesidir, her bir değer bir indis ile erişilmektedir. Listeyi oluşturan değerlere öğeler denilmektedir. Listeler karakter dizilerine, karakterlerden oluşan sıralı kümeler, benzer, tek farkı listelerin öğeleri herhangi bir tipte olabilir. Listeler ve karakter dizileri – ve sıralı küme şeklinde davranan diğer şeyler – diziler (sequences) adını almaktadır.

https://sonsuzus.github.io/posts/python-programlama-ders-09/

**Comment by sonsuzus** [comment]
Python listeler.
